### ISSUE TITLE
important update!

### ISSUE THREAD

**Issue by bruvv** [kevinlekiller/Newznab-Blacklist#11]
I noticed that it was blocking HDDVD releases (wich are 720p releases) and also BRRIP releases! 

I changed the dvd blacklist to:

```
[\\.-](dvd([-]?r(ip)?)?)[\\.-]
```

Now it's gone (thanks to CJ!)

Same goes for the xvid stuff

```
((xvid|divx)(?!hd)|(320|480)p)
```

**Comment by bruvv** [comment]
Correct, the xvid regex isn't working. Can someone look at it ?

edit:
Ignor this, it's working.

**Comment by thezoggy** [comment]
you have some test data on what its blocking that it shouldnt?

**Comment by bruvv** [comment]
I tested this with remove releases that were changed by update_parsing.

The first old is matching only DVD in it, the older one was also matching HDDVD (720 releases) BRRIP (720 and 1080p) releases. For example:

```
Galapagos.2006.E02.1080p.HDDVD.x264.PROPER-CtrlHD
```

(for your information, the old regex: `dvd|dvdrip|dvdr|dvd-r|dvd-rip` )
The new one isn't blocking that.

And for xvid one, it is working I just forgot a ( in the admin portal ;)
The second one (the old one, `xvid|divx|480p|320p` ) was also matching releases like:

```
Hamlet 1996 BRRip XvidHD 720p-NPW
```

Which is (how ever you want to say it) a HD release ;)

```
Width x Height  1280x576
```

An other problem:

```
nintendo|wii|ps3|xbox|nds|psp
```

is matching:
Friends - Season 3 (notice the NDS)
